homework_26_11_2/src/Car.java

```diff
@@ -0,0 +1,71 @@
+public class Car {
+    int maxSpeed;
+    float fuelAmount;
+    String productionYear;
+    float price;
+    String brand;
+    int currentSpeed;
+    boolean isEngineStarted;
+
+    public Car(int maxSpeed, float fuelAmount, String productionYear, float price, String brand, int currentSpeed, boolean isEngineStarted) {
+        this.maxSpeed = maxSpeed;
+        this.fuelAmount = fuelAmount;
+        this.productionYear = productionYear;
+        this.price = price;
+        this.brand = brand;
+        this.currentSpeed = currentSpeed;
```

Comment:
you don't need to set current speed, it's 0 after creation

Reply:
corrected